civihr_employee_portal/civihr_employee_portal.install

```diff
@@ -276,6 +276,15 @@ function civihr_employee_portal_update_7016() {
   drupal_write_record('webform_civicrm_forms', $civicrmWebform, 'nid');
 }
 
+/**
+ * Refreshes the node export files since the onboarding form file upload limit
+ * was increased.
+ */
+function civihr_employee_portal_update_7017() {
+  civicrm_initialize();
```

Comment:
Do we need to initialize civi?

Reply:
Yep, I tried it without it and only Drupal is bootstrapped in these upgraders. It fails without this with an error about not finding `civicrm_api3`

Reply:
I asked because the `drush_civihr_employee_portal_refresh_node_export_files` doen't seem like it needed it at first glsnce. Since its needed, will approve then